CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## NEXT
 
+## 1.8.3
+
+* Fixed a bug that caused shipping address to be queried for orders that contained only virtual products
```

Comment:
Both of these look good to me, but I think they are describing the same bug.

Reply:
d'oh... looking at the m2 changelog while copy-pasta'ing into m1's changelog.